src/components/common/form/message-item-types/DatesMoved.spec.jsx

```diff
@@ -0,0 +1,48 @@
+import { screen, fireEvent, waitFor } from '@testing-library/react';
+import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
+import DoubleDateMoved from './DatesMoved';
```

Comment:
Minor but could we change this so the import is DatesMoved?

Reply:
And similar with the describe block below

Reply:
Updated typo to be inline with component name

Reply:
Could you also change the describe block?

Reply:
Updated